aether-kernel/aether/kernel/api/views.py

```diff
@@ -382,11 +382,12 @@ def put(self, request, *args, **kwargs):
     def bulk_update(self, request, *args, **kwargs):
         partial = kwargs.pop('partial', False)
         try:
-            # get instances
-            instance = [
-                get_object_or_404(models.Submission.objects.all(), pk=entry['id'])
-                for entry in request.data
-            ]
+            # get instances in single SQL call
+            instance = get_list_or_404(
+                models.Submission.objects.all(),
```

Comment:
```suggestion
                filter_by_realm(
                    request,
                    models.Submission.objects.all(),
                    self.mt_field,
                ),
```

we forgot to filter by current realm!!! :scream: :scream: :scream: